app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,27 @@
+<ul id="<%= dom_id recipe %>" class="list-group">
+  <li class="list-group-item">
+    <strong>Name:</strong>
+    <%= recipe.name %>
+  </li>
+
+  <li class="list-group-item">
+    <strong>Preparation time:</strong>
+    <%= recipe.preparation_time %>
+  </li>
+
+  <li class="list-group-item">
+    <strong>Cooking time:</strong>
+    <%= recipe.cooking_time %>
+  </li>
+
+  <li class="list-group-item">
+    <strong>Description:</strong>
+    <%= recipe.description %>
+  </li>
+
+  <li class="list-group-item">
+    <strong>Public:</strong>
+    <%= recipe.public %>
+  </li>
+
+</ul>
```

Comment:
- [ ] I suggest that the index view should matches the wireframe from the sneak peek.
To achieve that kindly use the bootstrap card component.
## Your view
![image](https://user-images.githubusercontent.com/68030297/217751351-a0f0895e-77c5-4d0f-bdb8-f1307dd0fa02.png)

## The Wireframe 
![image](https://user-images.githubusercontent.com/68030297/217751477-9367ac1f-da8e-41de-900a-1a64dcf41ac8.png)